src/seeder.js

```diff
@@ -30,8 +30,15 @@ export default class Seeder {
         this.store = new Corestore(config.get('store.path'))
         await this.store.ready()
 
+        // Do we have a seed defined for the hyperswarm
+        const opts = {}
+        const seed = config.get('hyperswarm.seed')
+        if (seed !== '') {
```

Comment:
What if `config.seed` is undefined ... not that it is relevant for our case, but still `typeof seed === 'string' && seed.length === 64` might be safer

Reply:
It's not. It's defined in config/default.json

Reply:
Might be - it's not user supplied data though - it's part of the apps config. I'll add it anyway though as it can't hurt